docs/ext/mpd.rst

```diff
@@ -73,6 +74,8 @@ See :ref:`config` for general help on configuring Mopidy.
         Listens on all IPv4 interfaces
     ``::``
         Listens on all interfaces, both IPv4 and IPv6
+    ``~/.mopidy/socket``
+        Listen on the socket in your configuration directory
```

Comment:
Maybe something more generic here? For example: `/path/to/unix/socket.sock` ? And adjust the wording appropriately.